pennylane/devices/default_qubit_jax.py

```diff
@@ -27,6 +28,7 @@
     from jax.config import config as jax_config
     from jax.experimental.ode import odeint
 
+
 except ImportError as e:  # pragma: no cover
     raise ImportError("default.qubit.jax device requires installing jax>0.2.0") from e
```

Comment:
jax sparse stuff is much, much more recent. You should surely ask jax>0.3.20 which was released in September

Reply:
(jax sparse was around a bit before 0.3.20 but was very buggy)

Reply:
Done! 👍